src/test/groovy/readme/ReadmeExamples.java

```diff
@@ -184,8 +184,8 @@ void executionStrategies() {
 
         GraphQL graphQL = GraphQL.newGraphQL(StarWarsSchema.starWarsSchema)
                 .queryExecutionStrategy(new ExecutorServiceExecutionStrategy(threadPoolExecutor))
-                .mutationExecutionStrategy(new SimpleExecutionStrategy())
-                .subscriptionExecutionStrategy(new SimpleExecutionStrategy())
+                .mutationExecutionStrategy(new AsyncExecutionStrategy())
+                .subscriptionExecutionStrategy(new AsyncExecutionStrategy())
                 .build();
 
     }
```

Comment:
Do we have a test some where that tests how a mutation switches back to query strategy once the first level is resolved?

This will be more important with async if we get it wrong